### PR TITLE
GTFS FLEX Support

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
This PR adjusts various packages to fix bugs and render GTFS FLEX routes and trips correctly, and with warnings and notices where appropriate.

The snapshots in this PR depend on core-utils updates which are in a separate PR #300 .

### CODE REVIEW

**Comment by miles-grant-ibigroup** [comment]
Now that #300 has a package, this PR is ready for review @binh-dam-ibigroup

File: packages/trip-details/src/index.js [hunk 1]
```diff
@@ -192,6 +214,46 @@ function TripDetails({
...
+        {containsFlex && (
+          <TripDetail
+            icon={<Route size={17} />}
+            summary="This trip includes flexible routes."
+          />
+        )}
+        {pickupBookingInfo &&
+          pickupBookingInfo.map(info => (
+            <TripDetail
+              key={info.pickupMessage}
```
Comment (binh-dam-ibigroup) [inline packages/trip-details/src/index.js:226]: Sort props.

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM, some minor sorting comments.

File: packages/itinerary-body/src/styled.js [hunk 2]
```diff
@@ -169,6 +169,11 @@ export const AgencyInfo = styled.div`
   }
 `;
 
+export const CallAheadWarning = styled.div`
+  margin-top: 5px;
+  color: #b22727;
```
[the remark this answers is not in the document]
Comment (miles-grant-ibigroup) [reply packages/itinerary-body/src/styled.js]: Sorted in 5d40040651a55901e69f8a04401a657153658723

File: packages/trip-details/src/index.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply packages/trip-details/src/index.js:226]: Sorted in 5d40040651a55901e69f8a04401a657153658723

File: packages/trip-details/src/index.js [hunk 3]
```diff
@@ -5,6 +5,9 @@ import React from "react";
 import { CalendarAlt } from "@styled-icons/fa-solid/CalendarAlt";
 import { Heartbeat } from "@styled-icons/fa-solid/Heartbeat";
 import { MoneyBillAlt } from "@styled-icons/fa-solid/MoneyBillAlt";
+import { PhoneVolume } from "@styled-icons/fa-solid/PhoneVolume";
```
[the remark this answers is not in the document]
Comment (miles-grant-ibigroup) [reply packages/trip-details/src/index.js:6]: Sorted in 5d40040651a55901e69f8a04401a657153658723

File: packages/trip-details/src/index.js [hunk 4]
```diff
@@ -192,6 +214,46 @@ function TripDetails({
...
+              summary={`To take this route, ${callString(info)} ${advanceString(
+                info
+              )}.`}
+              description={info.pickupMessage}
+            />
+          ))}
+        {dropOffBookingInfo &&
+          dropOffBookingInfo.map(info => (
+            <TripDetail
+              key={info.dropOffMessage}
```
[the remark this answers is not in the document]
Comment (miles-grant-ibigroup) [reply packages/trip-details/src/index.js]: Sorted in 5d40040651a55901e69f8a04401a657153658723

**Review by philip-cline — APPROVED** [review]
Looks great to me!

**Pull request merged**